hudi-flink/pom.xml

```diff
@@ -157,7 +157,7 @@
     <dependency>
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
-      <scope>compile</scope>
+      <scope>provided</scope>
```

Comment:
same questions on the need to repeat this again in the child modules

Reply:
It's redundant, but do no harm.